system/k3s/k3s.SlackBuild

```diff
@@ -60,6 +60,21 @@ else
   exit 1
 fi
 
+bailout() {
+  echo "  Better to have a k3s group to run this script. "
+  echo "  create k3s group and add user to avoid always use of sudo: "
+  echo "    # groupadd k3s "
+  echo "    # chown root:k3s /etc/rancher/k3s/k3s.yaml "
+  echo "    # chmod 740 /etc/rancher/k3s/k3s.yaml "
+  echo "    # usermod -aG k3s \$USER (Replace \$USER with your username) "
+  echo "  Then restart k3s daemon. "
+}
+
+# Bail if group isn't valid on your system.
+if ! grep -q "^$PRGNAM:" /etc/group; then
+  bailout
```

Comment:
This needs an `exit 1` or something here or in the function to actually "bailout"

Reply:
Once you have the gid, the instructions for the group need to go in README or README.Slackware

Reply:
@aclemons But doesn't basically let to failure of build in sbo-bot?
I mean, I want to know if it's handled.